rules/rules-reviewed/azure/azure-aws-config.windup.xml

```diff
@@ -217,7 +235,68 @@
                     </message>
                     <link title="Azure Service Bus documentation" href="https://learn.microsoft.com/azure/service-bus-messaging"/>
                     <link title="Azure Service Bus libraries for Java" href="https://learn.microsoft.com/java/api/overview/azure/service-bus"/>
-                    <tag>servicebus</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="azure-aws-config-sqs-04002">
+            <when>                 
+                <project>
+                    <artifact groupId="software.amazon.awssdk" artifactId="sqs"/>
+                </project>
+            </when>
+            <perform>
+                <hint title="Amazon Simple Queue Service configuration" category-id="mandatory" effort="3">
```

Comment:
@KaiqianYang ,  I noticed that the newly added rules have the same hint title. Should this be same, or would it be better to make each title unique to avoid confusion?